src/main/webapp/js/email-browser.js

```diff
@@ -19,6 +19,74 @@
 var jasig = jasig || {};
 
 (function($, fluid) {
+
+	var ischecked = 0;
```

Comment:
👍 for checkboxCounter + a comment on checkboxes Global Checkness State with [google style guidlines](https://google.github.io/styleguide/javascriptguide.xml?showone=Comments#Comments)